src/authentication.ts

```diff
@@ -41,6 +42,37 @@ export async function redirectToLogin(config: TInternalConfig, customState?: str
   })
 }
 
+export async function redirectToRegister(config: TInternalConfig, customState?: string): Promise<void> {
```

Comment:
This is a complete copy of the `redirectToLogin` function, except for the redirection url? If so, I think we can do better than just duplicating the code.